lib/MusicBrainz/Server/Data/Event.pm

```diff
@@ -46,6 +46,8 @@ sub _columns
            'event.comment, event.last_updated';
 }
 
+sub _area_cols { [qw( event_area.area )] }
```

Comment:
We have `_id_column` under this, so maybe `_area_columns` would be better? I don't see a huge use of shortening here. I understand we call it cols right now in the role, but maybe it's better to be clear than concise :)

Reply:
I see we already use `_area_cols` for some entities so this has to be named like this, but I'd still suggest changing it everywhere in a followup commit. AFAICT it's the only place we use `_cols` instead of `_columns` outside of a small use in `extract_pot_db`.